ngx-tools/testing/src/utilities/events/dispatch-keyboard-event.spec.ts

```diff
@@ -0,0 +1,21 @@
+import { KEYS } from '@terminus/ngx-tools/keycodes';
+
+import { dispatchKeyboardEvent } from './dispatch-keyboard-event';
+
+
+describe(`dispatchKeyboardEvent`, function() {
+  let nodeMock: Element;
+
+  beforeEach(() => {
+    nodeMock = window.document.createElement('div');
+    nodeMock.dispatchEvent = jest.fn();
+  });
+
+
+  test(`should do something`, () => {
+    dispatchKeyboardEvent(nodeMock, 'keydown', KEYS.A);
+
+    expect(nodeMock.dispatchEvent).toHaveBeenCalledWith(expect.any(Event));
```

Comment:
The contents of the event are tested in the spec for `createKeyboardEvent` so rather than duplicating that check here, we just make sure that the dispatching is happening here.